individual.html

```diff
@@ -1,716 +1,912 @@
+                                                    sed do eiusmod tempor
+                                                    incididunt ut labore et
+                                                    dolore magna aliqua. Ut enim
+                                                    ad minim veniam, quis
+                                                    nostrud exercitation ullamco
+                                                    laboris nis.
+                                                </p>
+                                            </div>
+                                            <div class="card-design__logo">
+                                                <img
+                                                    src="./assets/Icons/logo-icon-credard.svg"
+                                                    alt="card-design-logo"
+                                                />
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
                     </div>
-                  </div>
-                </div>
-                <div
-                  class="card-design card__container--back saved-design__three"
-                >
-                  <div class="card__container__backContent">
-                    <div class="backContent--black"></div>
-                    <div class="backContent--white">
-                      <div class="backContent--white__logo"></div>
-                      <span>
-                        <p>Cvv</p>
-                      </span>
+                    <div class="user-info">
+                        <div class="user-info__img">
+                            <img
+                                src="./assets/images/user-profile.webp"
+                                alt="user-profile"
+                            />
+                            <p class="user-info-name">Designlover 123</p>
+                        </div>
+                        <div class="user-info__btns">
+                            <button id="openCommentModal">
+                                <img
+                                    src="./assets/Icons/comment-white.svg"
+                                    alt=""
+                                />
+                                <p>Comment</p>
+                            </button>
+                            <button>
+                                <img
+                                    src="./assets/Icons/like-white.svg"
+                                    alt=""
+                                />
+                                <p>Likes</p>
+                            </button>
+                            <button id="share-btn">
+                                <img src="./assets/Icons/share.svg" alt="" />
```

Comment:
Can you make it so the share-overlay doesn't cover the whole screen, if you play the figma prototype, you would get what I mean 
![image](https://user-images.githubusercontent.com/60205816/184091517-2e5a4562-6fa2-4728-aa1e-89daffec24ff.png)